tutorials/concur-calling-apis/concur-calling-apis.md

```diff
@@ -0,0 +1,76 @@
+---
+parser: v2
+author_name: Marc Bell
+author_profile: https://github.com/MarcBell-SAP
+auto_validation: true
+time: 20
+tags: [ tutorial>advanced, software-product>sap-concur]
+primary_tag: software-product>sap-concur
+---
+
+# Call Concur APIs using a Multiple Use Bearer Token.
```

Comment:
Thanks for the update.  You can remove the "." from the title.  Then I am ready to merge the request.